### ISSUE TITLE
Test the workflow

### ISSUE THREAD

**Issue by vecna** [tracking-exposed/pornhub.tracking.exposed#8]
As started by @spaghettinucleari 

---

[titolo]
AN ALGORITHMS DECIDES WHAT WILL TURN YOU ON

[contenuto]
Help us coming to a better understanding of what Pornhub is recommending you

Hello there!

Here tracking.exposed, our main objective is to put a spotlight on users’ tracking, profiling, on the data market and on the influence of algorithms. As long as these phenomena are shielded from view or understood only by experts, they cannot be tackled with the political determination that problems of such magnitude deserve. That is why we strive to explain the issue, test and promote new solutions, developed to benefit the community.

After the creation of facebook.tracking.exposed and youtube.tracking.exposed, a volunteer decided to respect the rules of the internet, in particular the number 34: There is porn of it. No exceptions.

So we therefore decided to analyze the online porn giant Pornhub, trying to understand how the profiling implemented by this company actually works. To do it better, we need your help, join the experiment.

Our browser extension is pornhub.tracking.exposed (download link), it collects and allow us to compare the individualized experiences that Pornhub users receive.

We ask you to install it and have a look at the best video we founded on Pornhub after months of exausting researches, don't worry your data will be completely anonymized. Learn more [here].

After installing poTREX, watch the best video we founded on the platform, don't forget to check on your personalized recommended videos at https://www.pornhub.com/recommended.

-https://www.pornhub.com/view_video.php?viewkey=ph585e7a92aaf77
-https://www.pornhub.com/view_video.php?viewkey=ph56f0416adbe1e
-https://www.pornhub.com/view_video.php?viewkey=ph5b96b786942d9
-https://www.pornhub.com/view_video.php?viewkey=ph55b2ec08ad5b1
-https://www.pornhub.com/view_video.php?viewkey=1327473218
-https://www.pornhub.com/view_video.php?viewkey=1636642701

DON'T FORGET TO CHECK YOUR RECOMMENDED AS WELL https://www.pornhub.com/recommended

Thank you 

pornaut

**Comment by vecna** [comment]
- [ ] - check the monitor interface, try to improve it
- [ ] - test how 3-5 people looks like if they watch the same video at the same time
- [ ] - get the test above review and summarized by 2-3-4 different humans?
- [ ] - enumerate the list of reddit channel
- [ ] - the text should be a facebook event on when the test will happen
- [ ] - we can announce (reddit+twitter+fbtrex) a 48 hours anticipation before the test

**Comment by vecna** [comment]
# Please thumb up or down:

- 6 of January the test, 4th the announcement. before we can improve the workflow

**Comment by spaghettinucleari** [comment]
test postponed to Jan 19th 2020

**Comment by spaghettinucleari** [comment]
*Communication strategy* 
by Monday 13th we should have:

- [ ] page potest/1 ready

- [ ] facebook event

- [ ] post on mastodon

- [ ] post on twitter

- [ ] post on reddit r/privacy, r/datascience, r/statistics, r/data, r/datasets, [r/database, r/cyberlaws]

**Comment by vecna** [comment]
https://www.reddit.com/r/privacy/comments/equgcy/on_sunday_january_19th_2020_join_the_first/
